Healthcare-Hospital-Management-System/Healthcare-Hospital-Management-System/Models/Doctor.cs

```diff
@@ -0,0 +1,21 @@
+
+namespace HealthcareHospitalManagementSystem.Models
+{
+    public class Doctor : Person
+    {
+        public string Specialization { get; set; }
+        public string Department { get; set; }
+
+        // Override the abstract method from Person
+        public override void DisplayInfo()
+        {
+            System.Console.WriteLine($"Doctor {Name}, Age: {Age}, Specialization: {Specialization}, Department: {Department}");
+        }
+
+        // Override the virtual method from Person
+        public override void Work()
+        {
+            System.Console.WriteLine($"Doctor {Name} is working with patients.");
```

Comment:
Console.WriteLine for WebAPI projects is not informative, all the info must be in the response models. Please make adjustments.